Makefile

```
@@ -0,0 +1,12 @@
+fmt:
+	cd code && cargo +nightly fmt
```

We currently format the code with stable rustfmt

```suggestion
	cd code && cargo fmt --all
```